src/views/Home/View.tsx

```diff
@@ -15,17 +15,21 @@ const View: React.FC = () => (
   <div className="home-page">
     <TypedHomePageQuery alwaysRender displayLoader={false} errorPolicy="all">
       {({ data, loading }) => {
-        if (canDisplay(data)) {
+        if (canDisplay(data) || true) {
```

Comment:
if that really was the intention here, `if` can be removed as the statement content will always be truthy.

Reply:
no... its the mistake, i'll change it